File: java/src/MainBatallaZapatoGallotronix.java

import model.batalla.Batalla;
import model.chimpos.Gallotronix;
import model.chimpos.*;
import model.logger.Logger;
import model.logger.Nivel;
import model.referi.Referi;
import model.tipo.TipoChinpokomon;

public class MainBatallaZapatoGallotronix {

    public static void main(String[] args) {
        Logger.getInstance(Nivel.INFO);
        Zapato zapato = new Zapato("Zapato", 5 , TipoChinpokomon.ROBOT);
        Gallotronix gallotronix = new Gallotronix("Gallotronix", 5, TipoChinpokomon.ANIMAL);
        Batalla batalla = new Batalla(zapato, gallotronix);
        Referi referi = new Referi(batalla);
        referi.gestionarBatalla();
    }

}

File: python/MainBatallaZapatoGallotronix.py
from model.logger.logger import Logger
from model.logger.nivel import Nivel
from model.chipos.Gallotronix import Gallotronix
from model.chipos.Zapato import Zapato
from model.chipos.enums import TiposDeChipos
from model.batalla.batalla import Batalla
from model.referi.referi import Referi

Logger.instance(Nivel.INFO)
zapato1 = Zapato("Zapato", 6, TiposDeChipos.ROBOT)
gallotronix = Gallotronix("Zapato 2", 5, TiposDeChipos.ANIMAL)
batalla = Batalla(zapato1, gallotronix)
referi = Referi(batalla)
referi.gestionar_batalla()
